keras_cv/models/segmentation/deeplab.py

```diff
@@ -56,13 +64,19 @@ def __init__(
         classes,
         include_rescaling,
         backbone,
-        weights,
+        backbone_weights=None,
+        weights=None,
```

Comment:
do we want to provide default voc weights?

Reply:
I mean default to `voc`, given keras.application models default to `imagenet` - https://www.tensorflow.org/api_docs/python/tf/keras/applications/resnet50/ResNet50

Reply:
Gotcha. Currently, our backbones don't have default weights. I don't personally love the idea of default weights, but I haven't thought about it a lot. Maybe we should discuss this as a package-wide option in our KerasCV sync?

Regardless of whether we include default weights, I would think we should keep them consistent across our model offerings.

Reply:
sounds good to me -- we could make this change later